.travis.yml

```diff
@@ -0,0 +1,25 @@
+{
+  language: objective-c,
+  before_install: [
+    "brew update"
+  ],
+  install: [
+    "mkdir -p $(brew --repo)/Library/Taps/travis",
+    "ln -s $PWD $(brew --repo)/Library/Taps/travis/homebrew-moira",
+    "brew tap --repair"
+  ],
+  env: PACKAGE=moira,
+  script: [
+    brew audit $PACKAGE,
+    brew install -v $PACKAGE,
+    brew test $PACKAGE
+  ],
```

Comment:
`brew test-bot`

Reply:
I don't want to run their test bot which uses Jenkins; I want to use Travis CI

Reply:
Er, yes - sorry, I'm forgetting moira is not in homebrew or a homebrew tap.